src/main/java/jenkins/plugins/office365connector/Office365ConnectorWebhookNotifier.java

```diff
@@ -135,11 +135,11 @@ public void sendBuildCompleteNotification() {
     public void sendBuildMessage(StepParameters stepParameters) {
         Card card;
         if (StringUtils.isNotBlank(stepParameters.getMessage())) {
-            card = getCard(3, stepParameters);
+            card = createBuildMessageCard(stepParameters);
         } else if (StringUtils.equalsIgnoreCase(stepParameters.getStatus(), "started")) {
-            card = getCard(1);
+            card = createJobStartedCard();
         } else {
-            card = getCard(2);
+            card = createJobCompletedCard();
         }
         if (card == null) {
```

Comment:
Seems like findbugs is into something with the redudant `if (card == null)`

Reply:
Yes, smart tool 👍